### PR TITLE
[backend] fix bugs

### PR DESCRIPTION
**Pull request by phoenix-xhuang**
1. change gpu lock time to 3mins.
2. ignore gpu_info request in metrics.
3. move gpu lock before merging in training task.
4. add time_it util to controller and command

### CODE REVIEW

**Review by under-chaos — APPROVED** [review]

File: ymir/backend/src/ymir-controller/controller/utils/utils.py [hunk 1]
```diff
@@ -66,3 +69,15 @@ def annotation_format_str(format: backend_pb2.LabelFormat) -> str:
         backend_pb2.LabelFormat.LABEL_STUDIO_JSON: 'ls_json',
     }
     return format_enum_dict[format]
+
+
+def time_it(f: Callable) -> Callable:
```
Comment (fenrir-z) [inline ymir/backend/src/ymir-controller/controller/utils/utils.py:74]: https://docs.python.org/zh-cn/3/library/timeit.html
可否考虑用内置的？

File: ymir/command/mir/commands/base.py [hunk 2]
```diff
@@ -10,7 +12,10 @@ class BaseCommand(ABC):
     def __init__(self, args: Any):
         self.args = args
 
-    # public: abstract
+    @utils.time_it
+    def cmd_run(self) -> int:
```
Comment (fenrir-z) [inline ymir/command/mir/commands/base.py:16]: 直接在 def run 上面加个计时就可以了吧

Comment (phoenix-xhuang) [reply ymir/command/mir/commands/base.py:16]: 修改每个run吗？这样一是每个文件都要改动，二是以后新增加的命令还得记着加，感觉不太合适吧

File: ymir/backend/src/ymir-controller/controller/utils/utils.py (hunk 1 above)
Comment (phoenix-xhuang) [reply ymir/backend/src/ymir-controller/controller/utils/utils.py:74]: 看了官方的指导，有两个问题，一是传参比较麻烦，二是直接接管了返回值，我们要获取返回值会变得更困难
https://docs.python.org/zh-cn/3/library/timeit.html
https://stackoverflow.com/questions/5086430/how-to-pass-parameters-of-a-function-when-using-timeit-timer

**Review by fenrir-z — APPROVED** [review]

**Pull request merged**